jitsi-media-transform/src/main/kotlin/org/jitsi/nlj/dtls/DtlsConfig.kt

```diff
@@ -23,4 +24,16 @@ class DtlsConfig {
     val handshakeTimeout: Duration by config {
         "jmt.dtls.handshake-timeout".from(JitsiConfig.newConfig)
     }
+
+    val ciphersSuites: List<Int> by config {
```

Comment:
This should probably be named `cipherSuites` not `ciphersSuites`

Reply:
If the BC code wants this as an IntArray, can we represent it as one here, rather than calling `toIntArray` every time?  Or does the config code not handle `IntArray`?

Reply:
I don't think our config code supports arrays, so we'd lose the usual behavior (e.g. being able to reset for tests) unless we do something like this:
```
private val cipherSuitesProp: List<Int> by config { ... }
fun cipherSuites(): IntArray = cipherSuitesProp.toIntArray()
```

Reply:
That might be a bit cleaner API-wise?  Not big deal tho.